conduct/code_of_conduct.md

```diff
@@ -0,0 +1,92 @@
+
+1. **Be careful in the words that you choose**. Be kind to others. Do not insult or put down other community members. Harassment and other exclusionary behavior are not acceptable. This includes, but is not limited to:
+   * Violent threats or violent language directed against another person
+   * Discriminatory jokes and language
+   * Posting sexually explicit or violent material
+   * Posting (or threatening to post) other people's personally identifying information ("doxing")
+   * Personal insults, especially those using racist or sexist terms
+   * Unwelcome sexual attention
+   * Advocating for, or encouraging, any of the above behavior
+   * Repeated harassment of others. In general, if someone asks you to stop, then stop
+
+1. **Moderate your expectations**. Many in our community volunteer their time. They are probably not purposefully ignoring issues, refusing to engage in discussion, avoiding features, etc. but often just unavailable.
+
+1. **When we disagree, try to understand why**. Disagreements, both social and technical, happen all the time and Jupyter is no exception. It is important that we resolve disagreements and differing views constructively. Remember that we’re different. The strength of Jupyter comes from its varied community that includes people from a wide range of backgrounds. Different people have different perspectives on issues. Being unable to understand why someone holds a viewpoint doesn’t mean they’re wrong. Don’t forget that it is human to err and blaming each other doesn’t get us anywhere. Instead, focus on helping to resolve issues and learning from mistakes.
+
+1. **A simple apology can go a long way**.  It can often de-escalate a situation, and telling
+someone that you are sorry is act of empathy that doesn’t automatically imply an admission of guilt (Canada has turned [*this notion into law*](http://www.theloop.ca/canadians-love-to-say-sorry-so-much-we-had-to-make-this-law/)).
+
+
+## Reporting
+
+If someone makes you or any other contributor feel unsafe or unwelcome,
+please report this in a timely manner. Code of conduct violations reduce
+the value of the community for everyone and we take them seriously.
+
+You can file a report by emailing
+[*conduct@jupyter.org*](mailto:conduct@jupyter.org) or by filing out
+[this form](https://goo.gl/forms/sJzOIie3zde9M71T2). For more details or
+information on reporting in-person at an event, please see our Reporting
+Guidelines.
+
+The online form gives you the option to keep your report anonymous or request
+that we follow up with you directly. While we cannot follow up on an anonymous
+report, we will take appropriate action to prevent a recurrence.
+
```

Comment:
@ellisonbg I agree. We should be specific that the Code of Conduct committee will review the report and act on it as appropriate.